gst/datarepo/gstdatareposink.c

```diff
@@ -764,6 +719,29 @@ gst_data_repo_sink_write_json_meta_file (GstDataRepoSink * sink)
   return ret;
 }
 
+/**
+ * @brief Check if the tensors are static or not
+ */
+static void
+gst_data_repo_sink_is_static_tensors (GstDataRepoSink * sink)
```

Comment:
Function name mismatches with the usage.

If it's ```gst_data_repo_sink_is_static_tensors```, it looks like to be used:
```sink->is_static_tensors = gst_data_repo_sink_is_static_tensors(sink);```

For this usage, it can be renamed:
```gst_data_repo_sink_set_is_static_tensors (sink);```